dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -0,0 +1,159 @@
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { remSpace, textSans } from '@guardian/source-foundations';
+import {
+	SvgCheckmark,
+	SvgNotificationsOff,
+	SvgNotificationsOn,
+	SvgPlus,
+} from '@guardian/source-react-components';
+import { decidePalette } from '../lib/decidePalette';
+
+type IconProps = {
+	isFollowing: boolean;
+	format: ArticleFormat;
+	iconIsFollowing: EmotionJSX.Element;
+	iconIsNotFollowing: EmotionJSX.Element;
+};
+
+const FollowIcon = ({
+	isFollowing,
+	format,
+	iconIsFollowing,
+	iconIsNotFollowing,
+}: IconProps): EmotionJSX.Element => (
+	<div
+		css={css`
+			background-color: ${isFollowing
+				? decidePalette(format).fill.shareIcon
+				: 'white'};
+			height: 24px;
+			width: 24px;
+			margin: 0 0.3ch 0 0.5ch;
+			${isFollowing
+				? 'padding: 0.35ch 0 0 0.3ch;'
+				: 'padding: 0.25ch 0 0  0.2ch;'}
+			border-radius: 100%;
+			${!isFollowing && 'border: 1px solid;'}
+			border-color: ${!isFollowing &&
+			decidePalette(format).text.articleLink};
+
+			svg {
+				fill: ${isFollowing
+					? 'white'
```

Comment:
Struggling to find an existing `decidePalette' option for this. 

`decidePalette(format).background.article` defaults to `transparent`.

I could potentially add to `fill.followingIcon` strategty but a bit confusing if there is a better exisiting fill value to use.